gensim/models/ldaseqmodel.py

```diff
@@ -896,9 +895,9 @@ def sslm_counts_init(self, obs_variance, chain_variance, sstats):
         T = self.num_time_slices
 
         log_norm_counts = np.copy(sstats)
-        log_norm_counts = log_norm_counts / sum(log_norm_counts)
-        log_norm_counts = log_norm_counts + 1.0 / W
-        log_norm_counts = log_norm_counts / sum(log_norm_counts)
+        log_norm_counts /= sum(log_norm_counts)
```

Comment:
it's a quite suspicious place. `sum(log_norm_counts)` means `log_norm_counts.sum(axis=0)`, not number and it's not obvious. Better to avoid it.